docker-compose-elastic-cloud.yml

```diff
@@ -23,6 +23,11 @@ services:
       - ELASTIC_APM_SOURCE_LINES_SPAN_APP_FRAMES=5
       - ELASTIC_APM_SOURCE_LINES_ERROR_LIBRARY_FRAMES
       - ELASTIC_APM_SOURCE_LINES_SPAN_LIBRARY_FRAMES
+      - ELASTIC_APM_ENVIRONMENT=production
+      - ELASTIC_APM_ENABLE_LOG_CORRELATION=true
```

Comment:
I haven't seen this setting before. It's not a flag we have in the Python agent. Do other agents have this flag or where does it come from?

Reply:
😞 it is only for the Java Agent https://www.elastic.co/guide/en/apm/agent/java/current/config-logging.html#config-enable-log-correlation

Reply:
removed too